wandb/sdk/interface/artifacts.py

```diff
@@ -662,6 +662,21 @@ def save(self) -> None:
         """
         raise NotImplementedError
 
+    def link(self, registry_path: str, aliases: Optional[List[str]] = None) -> None:
```

Comment:
Another case of `registry` <> `portfolio` terminology to resolve

Reply:
changed to target_path in next commit